.rubocop.yml

```diff
@@ -78,43 +83,25 @@ Style/ClassAndModuleChildren:
 Metrics/ClassLength:
   Enabled: false
 
-Layout/IndentationWidth:
-  Enabled: false
-
 Layout/EndAlignment:
-  Enabled: false
-
-Layout/ElseAlignment:
-  Enabled: false
-
-Performance/ChainArrayAllocation:
-  Enabled: true
+  EnforcedStyleAlignWith: start_of_line
 
 Style/BlockDelimiters:
   Enabled: false
 
 Style/RedundantSelf:
   Enabled: false
 
-Style/GuardClause:
-  Enabled: false
-
 Style/SymbolArray:
   Enabled: false
 
 Style/WordArray:
   Enabled: false
 
-Style/ParallelAssignment:
-  Enabled: false
-
-Style/PreferredHashMethods:
-  EnforcedStyle: verbose
-
-Style/RedundantException:
-  Enabled: false
+Style/RaiseArgs:
+  EnforcedStyle: compact
 
-Style/FrozenStringLiteralComment:
```

Comment:
We don't know how we can handle this cop, this is why we disactivated it. Otherwise, all it does is to give the same warning for all files. If we can find a way to freeze the strings then we are all for doing it. But how?